tests/testsuite/rustflags.rs

```diff
@@ -1237,7 +1456,12 @@ fn env_rustflags_misspelled() {
     for cmd in &["check", "build", "run", "test", "bench"] {
         p.cargo(cmd)
             .env("RUST_FLAGS", "foo")
-            .with_stderr_contains("[WARNING] Cargo does not read `RUST_FLAGS` environment variable. Did you mean `RUSTFLAGS`?")
+            .with_stderr_data(
+                "\
+[WARNING] Cargo does not read `RUST_FLAGS` environment variable. Did you mean `RUSTFLAGS`?
+...
+",
+            )
```

Comment:
Ideally, it's better to use the specific output without globbing. However, the output varies depending on the cmd used. Alternatively, we could simply check it separately if necessary.